Sources/Apollo/HTTPNetworkTransport.swift

```diff
@@ -183,17 +203,47 @@ public class HTTPNetworkTransport {
         }
       } catch let parsingError {
         self.handleErrorOrRetry(operation: operation,
-                                error: parsingError,
-                                for: request,
-                                response: response,
-                                completionHandler: completionHandler)
+                                 files: files,
+                                 error: parsingError,
+                                 for: request,
+                                 response: response,
+                                 completionHandler: completionHandler)
       }
     }
     
     task.resume()
     
     return task
   }
+
+  private func handleGraphQLErrorsOrComplete<Operation>(operation: Operation,
+                                                        files: [GraphQLFile]?,
+                                                        response: GraphQLResponse<Operation>,
+                                                        completionHandler: @escaping (_ result: Result<GraphQLResponse<Operation>, Error>) -> Void) throws {
+    guard let delegate = self.delegate as? HTTPNetworkTransportGraphQLErrorDelegate,
+      let graphQLErrors = response.parseErrorsOnlyFast(),
+      !graphQLErrors.isEmpty else {
```

Comment:
this is more of a note to self than to you: I need to add my readability `isNotEmpty` boolean extension to this code 🙃

Reply:
Done in #872